src/pages/singleDirection.tsx

```diff
@@ -92,7 +95,9 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{}> => {
             ) === index,
     );
 
-    return { props: { operator: operatorInfo.operator, lineName, service } };
+    return {
+        props: { operator: operatorInfo.operator, passengerType: passengerTypeInfo.passengerType, lineName, service },
```

Comment:
similar to  destructure above

Reply:
Added destructuring for 'passengerType'. Did the same for 'operator' for consistency's sake.